Marlin/src/feature/mmu3/mmu2_reporting.cpp

```diff
@@ -0,0 +1,794 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2024 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * mmu2_reporting.cpp
+ */
+
+#include <avr/pgmspace.h>
```

Comment:
I am having issues with this entire PR. It will not compile for non-AVR platforms.
I am trying to compile for board `BOARD_MKS_SGEN_L_V2` (`LPC1769`)

Reply:
Hopefully this is resolved by now through our helpful support network (?) If not, we'll have to take a closer look.

Reply:
Lots of stuff got buried:

<img width="227" alt="image" src="https://github.com/MarlinFirmware/Marlin/assets/13375512/73cd365e-ffac-4cbd-a0cf-955c6f08a128">

Reply:
This is resolved.